libs/xb/xs_ring_stubs.c

```diff
@@ -44,7 +44,7 @@ CAMLprim value ml_interface_read(value ml_interface,
 	CAMLlocal1(ml_result);
 
 	struct mmap_interface *interface = GET_C_STRUCT(ml_interface);
-	char *buffer = String_val(ml_buffer);
+	char *buffer = (char*) String_val(ml_buffer);
```

Comment:
This is already Bytes_val upstream

Reply:
```
mlvalues.h:#define String_val(x) ((const char *) Bp_val(x))
```
Does this suggest it's actually a `const` that is missing?